docs/modules/ROOT/pages/index.adoc

```diff
@@ -498,21 +508,186 @@ In each of the above sections, the following changes MUST be applied:
 
 == Trust model
 
-IMPORTANT: This section augments link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_trust_model++[Section 14, “Trust model,”] of the C2PA spec by adding additional trust signals related to the identity of _<<_actor,actors>>_ involved in creation of a _<<C2PA asset>>._ It does not replace any portion of the C2PA trust model.
+IMPORTANT: This section augments link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_trust_model++[Section 14, “Trust model,”] of the C2PA technical specification by adding additional trust signals related to the identity of _<<_actor,actors>>_ involved in creation of a _<<C2PA asset>>._ It does not replace any portion of the C2PA trust model.
 
-[#define-trust-model]
-[NOTE]
-.TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/22[issue #22]): Define the trust model.
-====
-Add language here that clarifies that the *<<_identity_assertion,identity assertion>>* introduces a second trust triangle, distinct from the trust triangle including the _claim generator,_ that enables the C2PA Manifest Consumer to make trust decisions regarding the _actor_ whose credential was used to generate the signature in this *<<_identity_assertion,identity assertion>>.*
+=== Digital trust introduction
+
+_This section is non-normative._
 
-Describe the trust signals that are intended to be conveyed by this assertion. Adapt from or possibly reference the following documents:
+Digital trust is typically described using a _trust triangle_ as shown in the following diagram:
 
-* link:++https://c2pa.org/specifications/specifications/1.3/guidance/Guidance.html#_trust_model++[6.3. Trust model] in C2PA implementers guidance (version 1.3)
-* link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_trust_model++[14. Trust model] in C2PA technical specification
-* link:++https://c2pa.org/specifications/specifications/1.0/security/Security_Considerations.html#_trust_model++[2.2 Trust model] in C2PA security considerations
-* link:++https://www.w3.org/TR/vc-data-model/#ecosystem-overview++[1.2 Ecosystem Overview] in W3C verifiable credentials spec
-* link:++https://www.w3.org/TR/vc-data-model/#use-cases-and-requirements++[1.3 Use Cases and Requirements] in W3C verifiable credentials spec
-====
+.Basic trust triangle
+image::trust-model/basic-trust-triangle.drawio.svg[Basic trust triangle,width=400,height=130,align="center"]
+
+The three roles depicted can each be performed by a human, organization, machine, or some combination thereof. A _credential holder_ establishes a relationship with a _credential issuer._ If the issuer trusts the credential holder, it will then issue a digital credential which makes statements about the credential holder and is signed by the issuer.
```

Comment:
Be clear about distinction btw subject and holder. Reference W3C discussion.

In basic trust triangle discussion, need to restate to say "verifies identity of subject, issues credential to holder."